src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
```

Comment:
```suggestion
    this.hidden = !this.hidden;
```
Метод должен менять значение на противоположное